vaadin-time-picker-flow/src/main/java/com/vaadin/flow/component/timepicker/GeneratedVaadinTimePicker.java

```diff
@@ -581,13 +603,32 @@ protected String getMaxString() {
      * <li>{@code hh:mm:ss.fff}</li>
      * </ul>
      *
+     * @deprecated use {@link #setMax(LocalTime)} instead.
+     *
      * @param max
      *            the String value to set
      */
+    @Deprecated
     protected void setMax(String max) {
         getElement().setProperty("max", max == null ? "" : max);
     }
 
+    /**
+     * <p>
+     * Description copied from corresponding location in WebComponent:
+     * </p>
+     * <p>
+     * Maximum time allowed.
+     * </p>
+     *
+     * @param max
+     *            the LocalTime value to set
+     */
+    protected void setMax(LocalTime max) {
```

Comment:
You should not make any changes to `GeneratedVaadinTimePicker`. It is auto-generated from the web component APIs, and if we later re-generate it based on a new version, we will lose all the changes that were made manually. This class's purpose is just to provide some auto-generated type-safe helpers for setting web component properties etc. which can optionally be utilized in the actual component class.

Reply:
Sorry. I don't that. Fixed it. Thanks